async function main() {
    console.log('Jejox')
}

main().catch(err => console.error(err))